instant/flags.vim

```diff
@@ -51,3 +51,11 @@ call s:plugin.Flag('clang_format_style', 'file')
 " "goimports" (https://godoc.org/golang.org/x/tools/cmd/goimports) to
 " additionally adjust imports when formatting.
 call s:plugin.Flag('gofmt_executable', 'gofmt')
+
+""
+" The path to the js-beautify executable.
+call s:plugin.Flag('js_beautify_executable', 'js-beautify')
+
+""
+" Default formatter for javascript files, 'clang_format' or 'js_beautify'.
+call s:plugin.Flag('javascript_formatter', 'clang_format')
```

Comment:
I am torn about this flag.  If you have js-beautify installed but not clang-format installed, it would be nice if we automatically picked up the one that worked.  I don't know a good solution though.  I am somewhat tempted to just leave them both installed and ask people to override the default with b:codefmt_buffer and some kind of hook...

Anyone else have thoughts?

Can you merge https://github.com/google/vim-codefmt/pull/21 and add tests for js-beautify too?